com.unity.render-pipelines.universal/Shaders/Utils/ScreenSpaceShadows.shader

```diff
@@ -56,14 +52,11 @@ Shader "Hidden/Universal Render Pipeline/ScreenSpaceShadows"
             UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(input);
 
             float deviceDepth = SAMPLE_TEXTURE2D_X(_CameraDepthTexture, sampler_CameraDepthTexture, input.uv.xy).r;
-
-#if UNITY_REVERSED_Z
-            deviceDepth = 1 - deviceDepth;
+#if SHADER_API_GLCORE
```

Comment:
what about gles and WebGL? UNITY_REVERSED_Z is more generic regarding z direction in all platforms that don't reverse z.

Reply:
I'm not sure if I understand it fully, but I suppose indeed the 1 - deviceDepth should be removed (handled in matrix), then if we are doing the second part to convert from 0 - 1 to opengl -1 <-> 1 than I guess we should do that for all OpenGL platforms?

Reply:
It doesn't reverse depth, it broadens the range [-1, 1] in opengl series. that's why I use 'SHADER_API_GLCORE' instead of 'UNITY_REVERSED_Z'. I need to add other opengl series.

Reply:
I checked it again. it is better to use 'UNITY_REVERSED_Z' as you said :)